docs/extra.css

```diff
@@ -14,7 +14,6 @@
 
 .md-typeset code,
 .md-typeset pre {
-    white-space: pre-wrap;
```

Comment:
This is removed as it breaks the very wide output of the cache dump output into an unreadable format. This custom CSS was added some time ago [four years ago](https://github.com/pi-hole/docs/commit/959458a2a3f3a79da2c0758e6981f15f8d1e90e4) with no apparent reason (pinging @PromoFaux). I checked through the docs and did not find anything where this did something bad.

Reply:
I cannot remember last week, let alone 4 years ago!

Reply:
I assumed so. I figured it wouldn't hurt to ask, though.